metadata/location/utah_address_points

```diff
@@ -0,0 +1,100 @@
+
+## Description
+
+### What is the dataset
+
+This dataset contains vector point data for over one million addresses in the State of Utah. Maintaining this dataset is an ongoing effort between UGRC, counties, and local municipalities. UGRC works with each county or municipality's MAL authority to continually improve the address point data.
+
+### What is the purpose of the dataset
+
+The Address Points GIS data are used for geocoding, 911 response, and analysis and planning purposes. The data are also used as reference data for the [api.mapserv.utah.gov](https://api.mapserv.utah.gov/) geocoding endpoint, and you can find the address points in many [web mapping applications](https://atlas.utah.gov/).
+
+### What does the dataset represent
+
+The address points in this dataset include information such as:
+
+- Full street address (i.e., the USPS mailing address, if the address is for a physical location [rather than a PO box])
+- Whether the location is a building
+- The type of unit
+- The city and ZIP code
+- Unique code identifiers, including the Federal Information Processing Standard ([FIPS](https://www.nist.gov/standardsgov/compliance-faqs-federal-information-processing-standards-fips)) county code and the US National Grid ([USNG](https://www.fgdc.gov/usng/how-to-read-usng)) spatial address
+- The date that the address point was added to the dataset
+
+### How was the dataset created
+
+The information in the Address Points dataset was originally compiled by Utah counties and municipalities and was aggregated by UGRC for the Master Address List ([MAL](https://docs.google.com/spreadsheets/d/1MIHEnmJ1dRekSd6n0s_oiS-Xy84JOaWUEqXXTnKsF1U/edit#gid=0)) grant initiative in 2012. The purpose of this initiative was to ensure that all state entities were using the same verified, accurate county and municipal address information.
+
+### How reliable and accurate is the dataset
+
+This dataset is a mapping-grade framework layer that receives regular updates. As with all our datasets, UGRC works to ensure the quality and accuracy of our data to the best of our abilities.
+
+## Credits
+
+### Data Source
+
+UGRC, local government partners
+
+### Host
+
+UGRC
+
+## Restrictions
+
+For more information on usage limitations for this dataset, please see our [disclaimer](https://gis.utah.gov/documentation/policy/license/#disclaimer).
+
+## License
+
+CC BY 4.0 License
+
+## Tags
+
+- UGRC
+- SGID
+- US Postal Service
+
+## Secondary Category
+
+## Data Page Link
+
+https://gis.utah.gov/products/sgid/location/address-points/
+
+## Updates
+
+### Update Schedule
+
+Counties have been placed on an update schedule depending on numerous factors, including rates of development and population growth. Densely populated counties, such as Salt Lake, Utah, and Weber, are updated monthly, while less populous counties may be updated quarterly or every six months.
```

Comment:
In the sgid index we are using semi annually as the term for six months

Reply:
I will add that in my style guide notes., thank you for catching that!